Articles of Association.md

```diff
@@ -0,0 +1,57 @@
+# Articles of Association
+1. Name
+    1. The name of the association is "Feather Minecraft Association"
+2. Purpose
+    1. The purpose of the association is
+        1. The promotion, development, maintenance, and distribution of the open-source project Feather and associated subprojects
+        2. The general support of open-source projects related to Minecraft
+3. Membership
+    1. Anyone whose interest is in accordance is with section 2 and have paid the membership fee is considered members of the association.
+    2. A membership is valid from the month in which the membership is paid and the following 11 months. 
+    3. The membership fee is determined by the General Assembly.
+    4. In the event of a serious violation of the Articles of Association or policies, or the event of direct opposition to the purpose stipulated in section 2. or function, the board may exclude a member if a majority of the board members are in favour. 
+        1. The member in question must be notified in writing that its exclusion has been proposed and a reason for this, no later than 14 days before the board meeting where the decision is made. The member has the right to give an explanation to the board in writing, and speak at the board meeting before the vote. 
+4. General Assembly
+    1. The General Assembly is the highest authority of the association.
+    2. The Annual General Assembly is held electronicly once a year before the end of ______ and is convened with at least three weeks' notice to members.
+    3. The public is eligible to attend the General Assembly.
+    4. All members who are actively contributing towards the purpose stipulated in section 2 are eligible to vote.
+    5. The agenda for the annual General Assembly must contain at least the following items:
+        1. Election of the Speaker
+        2. Election of secretary
+        3. Determination of present members
+        4. Financial reporting
+        5. Consideration of proposals received
+        6. Determination of membership fee
+        7. Approval of the budget
+        8. Election of chairman
+        9. Election of deputy chairman
+        10. Election of 3-7 board members and 2 substitutes
```

Comment:
I am still not quite sure why we need 2 substitutes rather than just increasing the number of board members to 5-9 if really necessary

Reply:
Board members are active, while substitutes can stay inactive. Like, I don't want to be a board member, but if you guys a missing one, count me in. If someone leaves the board and we are stile above 5 substitutes can decline without requiring an extraordinary assembly.